sourcetracker/__init__.py

```diff
@@ -9,10 +9,9 @@
 
 from ._compare import compare_sinks, compare_sink_metrics
 from ._sourcetracker import gibbs
-from ._plot import plot_heatmap
 
 
 __version__ = '2.0.1-dev'
 _readme_url = "https://github.com/biota/sourcetracker2/blob/master/README.md"
 
-__all__ = ['compare_sinks', 'compare_sink_metrics', 'gibbs', 'plot_heatmap']
+__all__ = ['compare_sinks', 'compare_sink_metrics', 'gibbs']
```

Comment:
You don't need ST_graphs in the init? I am just confirming

Reply:
No, in fact if it is in there it triggers an error for not having plot_heatmap any longer.